code/demo/android-native/app/build.gradle

```diff
@@ -1,37 +1,59 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 28
+  buildToolsVersion = '30.0.2'
+  compileSdkVersion sdkVersion.toInteger()
 
-    defaultConfig {
-        applicationId "org.orxproject.orxtestnative"
-        minSdkVersion 19
-        targetSdkVersion 28
-        versionCode 1
-        versionName "1.0"
+  ndkVersion "21.3.6528147"
 
-        ndk{
-            abiFilters "x86", "armeabi-v7a", "arm64-v8a", "x86_64"
-        }
+  defaultConfig {
+    applicationId "org.orxproject.orxtestnative"
+    targetSdkVersion sdkVersion.toInteger()
+    versionCode 1
+    versionName "1.0"
 
+    minSdkVersion 17
```

Comment:
I don't recall, but I think if I put minSdk 19 there was a reason, I'm not sure about 17

Reply:
I believe this change was made 2 years ago by an external contributor, and the only comment going with it is:
```
Raise minimum android platform to API19. Use c++_shared STL runtime. It will be copied to final application automatically by gradle during linking.
```

Reply:
The developer is free to target 19 or 21+. I don't think there is a need to target anything greater than 17. Besides, the existing `.mk` files are mixing 10, 14 and 19 and what not. For any new native lib, I would strongly encourage either 17 or 19. The former lets you reach a few more million devices. The latest NDK supports it, so why not...

Reply:
Ok then